client/src/views/AboutPage.tsx

```diff
@@ -0,0 +1,220 @@
+                <RelativeWrapper>
+                  <ProfileImageBackground color="#0145AC" />
+                  <ProfileImageWrapper imageURL="william.png" />
+                </RelativeWrapper>
+                <Spacer height={4} />
+                <Name>
+                  <b>William</b>
+                </Name>
+                <Spacer height={4} />
+                <Name>Extension</Name>
+              </ProfileWrapper>
+            </PicturesRow>
+            <Spacer height={32} />
+            <PicturesRow>
+              <ProfileWrapper>
+                <RelativeWrapper>
+                  <ProfileImageBackground color="#00C2FF" />
+                  <ProfileImageWrapper imageURL="linda.png" />
+                </RelativeWrapper>
+                <Spacer height={4} />
+                <Name>
+                  <b>Linda</b>
+                </Name>
+                <Spacer height={4} />
+                <Name>Frontend</Name>
+              </ProfileWrapper>
+
+              <ProfileWrapper>
+                <RelativeWrapper>
+                  <ProfileImageBackground color="#00C2FF" />
+                  <ProfileImageWrapper imageURL="andrew.png" />
+                </RelativeWrapper>
+                <Spacer height={4} />
+                <Name>
+                  <b>Andrew</b>
+                </Name>
+                <Spacer height={4} />
+                <Name>Frontend</Name>
+              </ProfileWrapper>
+            </PicturesRow>
+            <Spacer height={32} />
+            <ContactPaper>
+              <Typography variant="h5" align="center">
+                <b>Contact</b>
+              </Typography>
+              <Spacer height={16} />
+              <Typography align="center">
+                Have feedback, questions, or concerns about Wave? Don’t hesitate
+                to reach out!
+              </Typography>
+              <Spacer height={32} />
+              <PrimaryButton href="mailto:uwaterloowave@gmail.com">
+                Email
+              </PrimaryButton>
+            </ContactPaper>
+            <Spacer height={32} />
+            <Name variant="h6">
+              <b>Contributors</b>
+            </Name>
+            <Spacer height={32} />
+            <PicturesRow>
+              <ProfileWrapper>
+                <RelativeWrapper>
+                  <ProfileImageBackground color="#00C2FF" />
+                  <ProfileImageWrapper imageURL="yiwei.png" />
+                </RelativeWrapper>
+                <Spacer height={4} />
+                <Name>
+                  <b>Yi Wei</b>
+                </Name>
+                <Spacer height={4} />
+                <Name>Backend</Name>
+              </ProfileWrapper>
+            </PicturesRow>
+          </MainWrapper>
+        </Container>
+      </WaterWrapper>
+    </>
+  )
+}
+
+const MainWrapper = styled.div`
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+`
+
+const WaterWrapper = styled.div`
```

Comment:
Is it possible to make that background a global or should we just have that variable in every file where we want this gradient to be

Reply:
You're right, we should add it as a global. Let's do this in future PRs